docs/guides/getting-started/setup/_fragments/_intro.mdx

```diff
@@ -13,7 +15,15 @@ Tauri is a framework to build desktop applications with any frontend framework a
 
 In the following, we will first scaffold the frontend, set up the Rust project, and lastly show you how to communicate between the two.
 
+:::tip `create-tauri-app`
+
+The easiest way to scaffold a new project is the [`create-tauri-app`](https://github.com/tauri-apps/create-tauri-app) utility. It has opiniated templates for vanilla HTML/CSS/JavaScript and many frontend frameworks like React, Vue, and Svelte.
```

Comment:
Add a mention for yew here as well.

Reply:
Is this alright? I removed Vue so that we have 1 big js framework, 1 small js framework and 1 rust framework

Reply:
as a Vue fan, that breaks my heart but sounds good